docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+        "search_pattern": {"text": "Search query", "order": 3},
+        "created": {"text": "Created", "order": 4},
+        "updated": {"text": "Updated", "order": 5},
+    }
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+#### Views registration
+
+In order to render views in `invenio-administration`, they must be registered from the module on which they are implemented.
+
+A typical flow of actions to register views for a module is as follows:
+
+1. Create views in the module.
+2. Configure views to match your needs.
+3. Register views in `invenio-administration` entry point group.
+
+In OAI-PMH sets example, views are registered in `invenio-administration` as individual entry points from `invenio-rdm-records`.
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_rdm_records_oai_list = invenio_rdm_records.administration.views.oai:OaiPmhListView
+    invenio_rdm_records_oai_edit = invenio_rdm_records.administration.views.oai:OaiPmhEditView
+    invenio_rdm_records_oai_create = invenio_rdm_records.administration.views.oai:OaiPmhCreateView
+    invenio_rdm_records_details = invenio_rdm_records.administration.views.oai:OaiPmhDetailView
+```
+
+### Create custom view
+
+A custom view can be created by inheriting directly from `AdminView`.
+#### Usage
+
+```python
+from invenio_administration.views.base import AdminView
+
+class MyCustomView(AdminView):
+    """My custom view."""
+
+    name = "customview"
+    category = "My category"
+    template = "invenio_administration/index.html"
+    url = None
+    menu_label = "Custom View"
+    icon = "user"
+```
+
+The class defined for the custom view must be registered as an entry point, as follows:
+
+```bash
+vim invenio-rdm-records/setup.cfg
+```
+
+```ini
+[options.entry_points]
+invenio_administration.views =
+    invenio_module_admin_custom_view = invenio_module.administration.views:MyCustomView
+```
+
+For the full attributes list and description visit [reference docs](../../reference/administration_reference.md)
+
+### Customisation: dashboard view
+
+The dashboard view can be configured by creating a view that inherits from the `AdminView` and updating the config variable `ADMINISTRATION_DASHBOARD_VIEW` with the new declared view.
+
+```python
+ADMINISTRATION_DASHBOARD_VIEW = (
+    "invenio_administration.views.dashboard.AdminDashboardView"
+)
+```
+
+If there is no need to provide any additional template context, dashboard view can be also overridden by adding new jinja template in your local instance, as follows: `templates/invenio_administration/index.html`. 
+
+### Customisation: jinja templates
+
+The views are defined by jinja templates, and this can be easily extendable and overridable by redefining their blocks.
+
+Depending on the desired view to overridden, the correct template should be extended:
```

Comment:
Can we rephrase? I am not understanding what I have to do here, and why I have to do it.